### PR TITLE
moved to native ed25519

### PR DESCRIPTION
**Pull request by ccorcoveanu**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,17 @@ module github.com/ElrondNetwork/elrond-deploy-go
 go 1.12
 
 require (
-	github.com/ElrondNetwork/elrond-go v1.0.77
```
Comment (ccorcoveanu) [inline go.mod]: After `EN-5789-from-kyber-to-go-ed25519` is merged, `go.mod` should be changed again to point to a release tag

File: cmd/filegen/main.go [hunk 2]
```diff
@@ -203,6 +198,7 @@ func generateFiles(ctx *cli.Context) error {
 		totalAddressesWithBalances = ctx.GlobalInt(numAddressesWithBalances.Name)
 	} else {
 		totalAddressesWithBalances = numOfShards*(numOfNodesPerShard+numOfObserversPerShard) + numOfMetachainNodes + numOfMetachainObservers
+		totalAddressesWithBalances = numOfShards*(numOfNodesPerShard+numOfObserversPerShard) + numOfMetachainNodes + numOfMetachainObservers
```
Comment (AdoAdoAdo) [inline cmd/filegen/main.go]: why is this doubled?

File: cmd/filegen/main.go [hunk 3]
```diff
@@ -556,17 +550,8 @@ func isMintValueValid(mintValue string) error {
...
-	switch consensusType {
-	case "bls":
-		suite = kyber.NewSuitePairingBn256()
-	case "bn":
-		suite = kyber.NewBlakeSHA256Ed25519()
-	default:
-		suite = nil
-	}
+func getNodesKeyGen() crypto.KeyGenerator {
+	suite := kyber.NewSuitePairingBn256()
```
Comment (AdoAdoAdo) [inline cmd/filegen/main.go]: please rebase as this was changed on master

File: cmd/filegen/main.go (hunk 2 above)
Comment (ccorcoveanu) [reply cmd/filegen/main.go]: I think I pressed cmd+D by mistake and did not see it. Fixed

File: cmd/filegen/main.go (hunk 3 above)
Comment (ccorcoveanu) [reply cmd/filegen/main.go]: done

**Pull request merged**